packages/app/src/app/components/dashboard/InputText.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { InputHTMLAttributes } from 'react';
+import styled from 'styled-components';
+import { Stack } from '@codesandbox/components';
+import { Label } from './Label';
+
+const StyledInput = styled.input`
+  padding: 12px 16px;
+  background-color: #2a2a2a;
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  color: #e5e5e5;
+  line-height: 24px;
+  border: none;
+  border-radius: 2px;
+
+  &:hover {
+    box-shadow: 0 0 0 2px #e5e5e51a;
+  }
+
+  &:focus {
+    outline: 1px solid #ac9cff;
+  }
+`;
+
+interface InputTextProps extends InputHTMLAttributes<HTMLInputElement> {
+  id: string;
+  label: string;
+  name: string;
+}
+
+export const InputText = ({
```

Comment:
Should this also handle validation to show ✅ or ❌ icons?

Reply:
Great question, and thanks for reminding me! In a proper design system I'd argue that it should be a different component as it has different requirements and functionality, and imo it is best to not create Swiss knife components. In this case it doesn't really matter but I'll see if it's easy to add - forgot about this.

Reply:
I've created [a separate Linear ticket](https://linear.app/codesandbox/issue/XTD-300/onboarding-questions-validation) for this.